using-datapower-in-kubernetes/getting-started-datapower-in-gke.md

```diff
@@ -0,0 +1,350 @@
+your application and is used to update your pods or replica sets to try to match the current state of your application to that of the desired state.
+You can specify deployments, pods, services, replica sets, etc. declaratively in Kubernetes by writing a
+corresponding configuration file.
+
+In a separate Google Cloud Shell session, we can run `$ kubectl describe deployments` to show the details
+ of the *datapower* deployment we just created.
+ 
+ ![kubectl describe deployments](media/kubectl_describe_deployments_datapower.png)
+
+Don't worry if not everything from the output makes sense at this point, the main idea here is that we can use the `describe` command to 
+show the details of various Kubernetes objects as we will soon see. Additionally, note the section titled "NewReplicaSet" which lists the replica set used by our deployment.
+We can see the details of our replica sets by using the `$ kubectl describe replicasets`.
+
+![kubectl describe replicasets](media/kubectl_describe_replicasets.png)
+
+As we can see in the section "Replicas", this replica set describes our current state of "1 current" running pod
+and our desired state of "1 desired" running pod.
+
+Similarly, we can run the `$ kubectl get pods` command to view the running datapower pod. 
+
+![kubectl get pods](media/kubectl_get_pods.png)
+
+To demonstrate the role of replica sets, lets manually delete the datapower pod.
+To do this, we run: 
+
+```
+$ kubectl delete pods <pod_name>
+```
+
+where <pod_name> is the name of the pod as listed in `$kubectl get pods`.
+
+We expect the replica set to note that change of state, our datapower pod being deleted, and promptly start a new pod for us running our datapower image.
+To ensure this, we check `$ kubectl get pods` again which shows a new pod (note the new name) running our datapower image.
+We can can also see this change in the event output of our replica set.
+
+```
+$ kubectl describe replicasets
+```
+
+Lastly, our first Google Cloud Session will also show our previous interactive session with the container has now
+been terminated since we issued a deletion of the pod. To reattach to the new container created by the replica set,
+we can issue:
+
+```
+$ kubectl attach <pod_name> --container="datapower" --stdin --tty
+```
+
+And log in to datapower using the default user "admin" and "admin" password.
+
+Once you are done you can delete the deployment as we will not make further use of it in this tutorial.
+
+We delete the deployment by issuing:
+`$ kubectl delete deployment <deployment_name>`
+
+## Deploying  a Composed Application in Kubernetes
+
+Now that we've gotten a small taste of Kubernetes, we can begin to deploy a simple composed applicaiton.
+For this tutorial, we will use a similar application to the one in the [Docker Compose sample](https://developer.ibm.com/datapower/2016/11/11/a-datapower-with-docker-compose-hello-world-application/).
```

Comment:
We'll have multiple Docker Compose samples. You may want to specify Hello World sample